source/main.cpp

```diff
@@ -39,6 +39,9 @@ using namespace Aws::Iot::DeviceClient::Logging;
 #if !defined(EXCLUDE_ST)
 using namespace Aws::Iot::DeviceClient::SecureTunneling;
 #endif
+#if !defined(EXCLUDE_FP)
+using namespace Aws::Iot::DeviceClient::FleetProvisioningNS;
```

Comment:
Can you update the namespace to FleetProvisioning and Class name to FleetProvisioningFeature to maintain naming standards across the program ?

Reply:
We don't want to change the class name to include feature because it doesn't implement the same feature interface.